payment/payment-kit/demoMiniApp-02/demoMiniNativeApp/src/main/java/vn/teko/hestia/android/demomininativeapp02/MiniAppActivity.kt

```diff
@@ -0,0 +1,70 @@
+package vn.teko.hestia.android.demomininativeapp02
+
+import android.os.Bundle
+import android.widget.Button
+import androidx.appcompat.app.AlertDialog
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.mini_app_activity_mini_app.*
+import vn.teko.android.payment.kit.PaymentKit
+import vn.teko.android.payment.kit.PaymentResultV2Callback
+import vn.teko.android.payment.kit.model.request.PaymentRequestBuilder
+import vn.teko.android.payment.kit.model.request.PaymentV2Request
+import vn.teko.android.payment.kit.model.result.PaymentV2Result
+
+class MiniAppActivity : AppCompatActivity() {
+
+    private val paymentCallbackV2: PaymentResultV2Callback by lazy {
+        object : PaymentResultV2Callback {
+            override fun onResult(result: PaymentV2Result) {
+                val paymentResultStatus = when (result.resultCode) {
+                    PaymentV2Result.RESULT_CANCELED -> "canceled"
+                    PaymentV2Result.RESULT_FAILED -> "failed"
+                    PaymentV2Result.RESULT_SUCCEEDED -> "succeeded"
+                    else -> "processed with unknown status"
+                }
+                val message = "Payment was $paymentResultStatus"
+
+                AlertDialog.Builder(this@MiniAppActivity)
+                    .setTitle(message)
+                    .setMessage(result.error?.message.orEmpty())
+                    .setPositiveButton("OK") { _, _ -> }
+                    .show()
+            }
+        }
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.mini_app_activity_mini_app)
+
+        title = "Mini Application - Host App Payment Flow"
+
+        findViewById<Button>(R.id.btnRequestPayment).setOnClickListener {
```

Comment:
chỗ này đang thấy bị crash do
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.Button.setOnClickListener(android.view.View$OnClickListener)' on a null object reference

Reply:
Do mình dùng 2 file layout giống cùng tên nên nó bị ghi đè và xảy ra lỗi ạ. Em đã đổi lại tên file layout rồi ạ.